### ISSUE TITLE
TypeError: Cannot read property 'data' of null

### ISSUE THREAD

**Issue by theorangepotato** [beakerbrowser/beaker#919]
**Operation System:** Debian 9 (Stretch)
**Beaker version:** Beaker 0.8.0-prerelease.3 (commit #f46ce3)

I have repeatedly received the following error (possibly related to Fritter?):

```
Error
A Javascript error occurred in the main process

Uncaught Exception:
TypeError: Cannot read property 'data' of null
    at Request.onvalue [as_callback] ($PATH_TO_BEAKER/app/node_modules/hypercore/lib/replicate.js:135:19)
    at Request.callback ($PATH_TO_BEAKER/app/node_modules/random-access-storage/index.js:150:8)
    at onread ($PATH_TO_BEAKER/app/node_modules/hyperdrive/node_modules/random-access-file/index.js:80:31)
    at FSReqWrap.wrapper [as oncomplete] (fs.js:658:17)
```

I have not noticed a pattern about when this error occurs yet, but I am usually using Fritter at the time (however, that could be a sampling bias). It creates a dialog box that will come back every time I click 'OK' until I kill Beaker.

**Comment by pfrazee** [comment]
Yikes, thanks for reporting, I'll check into that asap.

**Comment by pfrazee** [comment]
Fixed in hypercore 6.12.7, see https://github.com/mafintosh/hypercore/issues/146
